contrib/CMakeLists.txt

```diff
@@ -91,6 +91,7 @@ add_contrib (protobuf-cmake protobuf)
 add_contrib (openldap-cmake openldap)
 add_contrib (grpc-cmake grpc)
 add_contrib (msgpack-c-cmake msgpack-c)
+add_contrib (BLAKE3)
```

Comment:
`contrib/blake3` should not refer to your personal repository.
It's better to make it refer to the official blake3 repository, and then provide a custom `CMakeLists.txt` file in the separate directory blake3-cmake.